dice.js

```diff
@@ -0,0 +1,15 @@
+const rollDice = function(num = 2) {
+    if (num < 1) { num = 2 }
+    const num1 = Math.floor(Math.random() * 5) + 1

```

Comment:
This code will give a number between 1 and 5.
Use this to get a number between 1 and 6 ```  Math.floor(Math.random() * 6) + 1 ```
```Math.random``` will give random number between 0 and 0.99... , it will never give 1.
so here ```Math.random() * 6```  will give number between 0 and 5.99...  using ```Math.floor()``` will result to an integer between 0 and 5 and adding ```+1``` you will  end up  with an integer between 1 and 6.

to read more about ```Math.random()```:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random